tests/Unit/ItemTest.php

```diff
@@ -4312,6 +4312,93 @@ public static function getTitleDataProvider(): array
                 ,
                 'Item Title',
             ],
+            'Test RSS 0.91 XML and DTD' => [
+<<<XML
+<?xml version="1.0"?>
+<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
+<rss version="0.91">
+	<channel>
+		<item>
+			<title>Item Title</title>
+		</item>
+	</channel>
+</rss>
+XML
+                ,
+                'Item Title',
+            ],
+            'Test RSS 0.91 XML and DTD and external HTML entity' => [
```

Comment:
> HTML entity

The RSS 0.91 DTD actually copies the entity list from HTML 3.2 and appears to be a subset of  `declare_html_entities`.